server/src/main/java/org/elasticsearch/action/admin/cluster/coordination/ClusterFormationInfoRequestBuilder.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.cluster.coordination;
+
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.action.support.nodes.NodesOperationRequestBuilder;
+import org.elasticsearch.client.internal.ElasticsearchClient;
+
+public class ClusterFormationInfoRequestBuilder extends NodesOperationRequestBuilder<
```

Comment:
Likewise this doesn't seem necessary, you may as well just construct the request directly.